fca_import_scripts/lib/ext_to_sql.rb

```diff
@@ -127,12 +125,10 @@ def log(str, newline: true)
 
   def write_progress
     @i ||= 0
-    if (@i += 1) % 10_000 == 0
-      log '.', newline: false
-    end
+    log '.', newline: false if (@i += 1) % 10_000 == 0
```

Comment:
I'd say this is a good candidate for a guard clause as I described before — looks a bit noisy this way.